src/components/a-td/index.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { withBaseGlobalAndAllHocs } from '../../js/abstract/hocs';
+import defineOnce from '../../js/define-once';
+import alignPropType from '../../js/prop-types/align-prop-type';
+import floatPropType from '../../js/prop-types/float-prop-type';
+import statePropType from '../../js/prop-types/state-prop-type';
+// import the styles used for this component
+import styles from './index.scss';
+
+const TableCellBase = withBaseGlobalAndAllHocs(HTMLTableCellElement);
+
+class AXATd extends TableCellBase {
+  static tagName = 'axa-td'
+  static builtInTagName = 'td'
+
+  // specify runtime type-checking here, if you use custom attributes
+  // this will also derived your needed observed attributes automatically for you
+  static propTypes = {
+    classes: PropTypes.string,
+    action: PropTypes.bool,
+    align: alignPropType,
+    bold: PropTypes.bool,
+    dense: PropTypes.bool,
+    float: floatPropType,
+    state: statePropType,
+    strong: PropTypes.bool,
```

Comment:
we have also designs for `part` in sense of allocations in percent

<img width="783" alt="screen shot 2018-11-20 at 13 46 03" src="https://user-images.githubusercontent.com/914443/48774411-81cc5100-ecca-11e8-82de-31d778d77db0.png">